Dockerfile

```diff
@@ -1,22 +1,52 @@
-FROM gapsystem/gap-container
+FROM ubuntu:bionic
 
 MAINTAINER The GAP Group <support@gap-system.org>
 
 ENV DEBIAN_FRONTEND noninteractive
 
-# Prerequirements
-RUN    sudo apt-get update -qq \
-    && sudo ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime \
-    && sudo apt-get -qq install -y \
-                                libboost-dev \
-                                libgmp-dev libgmpxx4ldbl \
-                                libmpfr-dev libmpfi-dev libmpc-dev \
-                                autoconf autogen libtool libreadline6-dev libglpk-dev \
-                                libmpfr-dev libcdd-dev libntl-dev git polymake
+# Prerequisites
+RUN    dpkg --add-architecture i386 \
+    && apt-get update -qq \
+    && apt-get -qq install -y \
+            autoconf \
+            autogen \
+            automake \
+            build-essential \
+            cmake \
+            g++ \
+            gcc \
+            gcc-multilib \
+            git \
+            libboost-dev \
+            libcdd-dev \
+            libglpk-dev \
```

Comment:
@alex-konovalov Why is this here? Perhaps for @sebasguts GLPKInterface? But that doesn't ship with GAP, does it?

Reply:
So: Can I remove it?

Reply:
I think it can be removed, yes.